content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -323,14 +316,9 @@ Note that you can rename the [Prisma-level](prisma-schema/relations#relation-fie
 
 ## Introspection with an existing schema
 
-<Admonition type="warning">
-
-**MongoDB not supported** <br />
-Introspecting _repeatedly_ with an existing schema will lead to data loss in the schema file.
-
-</Admonition>
+Running `prisma db pull` for relational databases with an existing schema file merges manual changes made to the schema, with changes made in the database. (This functionality has been added for the first time with version 2.6.0.) For MongoDB, Introspection for now is meant to be done only once for the initial data model. Running it repeatedly will lead to loss of custom changes, as the ones listed below. <!-- TODO Remove when MongoDB Re-Introspection has been implemented -->
 
-In version 2.6.0 and later, running `prisma db pull` with an existing schema file merges manual changes made to the schema, with changes made in the database. Introspection maintains the following manual changes:
+Introspection for relational databases maintains the following manual changes:
```

Comment:
Explicit information about Re-Introspection not working for MongoDB and what that means.